antora-playbook-staging.yml

```diff
@@ -54,7 +54,7 @@ content:
     branches: [release/7.6, release/7.2, release/7.1, release/7.0, release/6.6, capella, elixir]
     start_path: docs
   - url: https://github.com/couchbasecloud/couchbase-cloud
```

Comment:
Doesn't the staging playbook need to update to   - url: https://github.com/couchbase/docs-capella as well?

Reply:
It's already there from an earlier commit (I've built Staging to test with this configuration)